### DISCUSSION TITLE
Cannot make changes modifiable is off when trying Code actions

### DISCUSSION THREAD

**Discussion by semi710** [mrcjkb/rustaceanvim#359] [Q&A] [answered]
![image](https://github.com/mrcjkb/rustaceanvim/assets/60490474/781277f3-7ae9-4fa7-a187-6adc9271c7ea)

**Comment by mrcjkb** [comment, marked as answer]
Hey :wave: 

The stack trace tells me this is caused by the nvim-markdown plugin.

**Reply by semi710** [reply]
Yep, disabling that plugin fixes the issue, but m unsure why this is happening.

**Reply by mrcjkb** [reply]
I would ask there or check the issue tracker. 
It looks like it thinks the code action window is a file and is trying to save it or something.
